app.js

```diff
@@ -88,7 +88,8 @@ app.use(helmet.contentSecurityPolicy({
   directives: {
     defaultSrc: [
       'www.youtube.com',
-      'https://public.etherpad-mozilla.org'
+      'https://public.etherpad-mozilla.org',
+      'https://pontoon.mozilla.org'
```

Comment:
Well... Its in an iframe and trying to load everything for users to translate the site and it needs to display everything correctly the same way on production.